pegasus/sites.v3/code.org/views/teach_ai_skinny_banner.haml

```diff
@@ -1,6 +1,6 @@
-= inline_css 'generated/homepage-teach-ai-banner.css'
+= inline_css 'generated/teach-ai-skinny-banner.css'
```

Comment:
Is this also supposed to be behind the DCDO flag?

Reply:
Nope! It used to be behind a flag but we removed it [here](https://github.com/code-dot-org/code-dot-org/pull/52464/files#diff-9d6502fdbc36cecfe029344c72579bfa407731327c21d8f5da94dee14289788b). With the move to the code.org/ai page it's ok to have it evergreen for now.

Reply:
Gotcha! Sounds good :)